src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,160 @@
+package subway;
+
+import io.restassured.path.json.JsonPath;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.extractableResponse.LineApiExtractableResponse.*;
+import static subway.extractableResponse.StationApiExtractableResponse.createStationByStationName;
+
+@DisplayName("지하철 노선 관련 기능")
+@Sql("/truncate.sql")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void 지하철_노선을_생성() {
+        // given
+        String lineName = "신분당선";
+        Long upStationId = createStationByStationName("강남역").jsonPath().getLong("id");
+        Long downStationId = createStationByStationName("신논현역").jsonPath().getLong("id");
+        Map<String, Object> requestBody = createRequestBody(lineName, "bg-red-600", upStationId, downStationId, 10);
+
+        // when
+        createLine(requestBody);
+
+        // then
+        List<String> lineNames =
+                selectLines().jsonPath().getList("name", String.class);
```

Comment:
```suggestion
    public static LineResponse selectLines2() {
        ExtractableResponse<Response> response = RestAssured
                .given().log().all()
                .when().get("/lines")
                .then().log().all()
                .statusCode(HttpStatus.OK.value())
                .extract();
        
        return response.as(LineResponse.class);
    }
```
selectLines와 같은 response 반환해주는 부분도 이렇게 써볼 수도 있어요!